content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Collector configuration best practices
+linkTitle: Collector config
+description: Best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When configuring the OpenTelemetry (OTel) Collector, consider the following
+practices to better secure your Collector instance.
+
+## Receivers and exporters
+
+We recommend enabling only the minimum required components for a Collector
+configuration: one receiver and one exporter. Configuring only the minimum set
+of components exposes a minimum set of required ports.
```

Comment:
"Ports" seem to be too specific, consider rephrasing it to "exposes a minimum attack surface" - it could be TCP/UDP ports, component dependency (SBOM), etc.

Reply:
Changed in cbf67a9. Thanks!